sessions/recap-project-5/art-gallery-app/README.md

```diff
@@ -1,41 +1,53 @@
+# Recap Project 5: Art Gallery
 
-There is an API that provides you with all art pieces to show in the gallery:
+In this project, your team will create an interactive art gallery app.
 
-```
-https://example-apis.vercel.app/api/art
-```
-
-Each item given by the API will include an URL to the image of the art piece. Use the `next/image` component to display the image. Don't forget to add the domain `example-apis.vercel.app` to the Next.js config file. Refer to the handout of the Session **Next.js Basics and Routing** for help (always restart the dev server after a config change).
+This is a group project, so please create **only one project per group** following the [guide in the template section](./README.md#template).
 
-Besides the art pieces themselves, your gallery app needs to handle data that changes on user interaction. You are required to establish a state to store additional information, that is linked to the art pieces given by the API. Please refer to the handout of the session **React Data Fetching** for guidance.
+## Overview
 
-The following user stories define the features you should add to the app. We advise you to work on the user stories roughly in the order they are noted here. Some user stories depend on each other. Some can be implemented independently.
+This project involves creating an art gallery app with four main pages: **Spotlight**, **Gallery**, **Favorite**, and **Detail**. Users will have the ability to like or unlike art pieces, and all liked pieces will appear on the **Favorite** page.
 
-Create a new repository on GitHub and add everybody in your group als collaborator. When starting with the app you should implement the basic setup in a mob. Afterwards you should split up. Each group member can contribute parts to the app individually or in pair programming. Use different branches to work in parallel.
+Each item from the API includes a URL to the art piece image. Use the `next/image` component to display these images. Remember to add `example-apis.vercel.app` to the allowed domains in the Next.js configuration file. You can refer to the **Next.js Basics and Routing** session handout for guidance. (Don’t forget to restart the development server after making any configuration changes.)
 
-Plan your work wisely and talk about your next steps in the team.
+In addition to displaying the art pieces, your gallery app should handle dynamic data that updates based on user interactions. You’ll need to establish a state to manage information linked to the art pieces from the API. Refer to the **React Data Fetching** session handout for further guidance.
 
-The goal of this recap project is not necessarily to finish all features, but to write clean code with isolated components. It is advised to follow the noted tasks in each user story.
+The goal of this project is to write clean code with well-isolated components rather than to complete all features. Follow the outlined tasks within each user story for structure.
```

Comment:
```suggestion
The goal of this project is to write clean code with well-isolated components rather than to complete all features. Follow the outlined tasks within each user story to get on track.
```